ui/src/status/fixtures.ts

```diff
@@ -1,9 +1,40 @@
 import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
 import {TEMP_VAR_DASHBOARD_TIME} from 'src/shared/constants'
+import {NEW_DEFAULT_DASHBOARD_CELL} from 'src/dashboards/constants/index'
+import {DEFAULT_AXIS} from 'src/dashboards/constants/cellEditor'
+import {Cell, CellQuery} from 'src/types'
+import {CellType} from 'src/types/dashboard'
 
-export const fixtureStatusPageCells = [
+const emptyQuery: CellQuery = {
+  query: '',
+  source: '',
+  queryConfig: {
+    database: '',
+    measurement: '',
+    retentionPolicy: '',
+    fields: [],
+    tags: {},
+    groupBy: {},
+    areTagsAccepted: false,
+    rawText: null,
+    range: null,
+  },
+}
+
+const emptyAxes = {
```

Comment:
would you consider typing this? for empty dashboards i made a dedicated type, but maybe there's an Axis type that this conforms to already?

Reply:
yup.